boards/arm/mec1501modular_assy6885/doc/index.rst

```diff
@@ -22,23 +22,31 @@ Hardware
 - 256 KB RAM and 64 KB boot ROM
 - GPIO headers
 - UART1 using microUSB
-- UART0 and UART2 exposed in headers
-- FAN0, FAN1, FAN2 headers
-- FAN PWM interface
-- JTAG/SWD, ETM and MCHP Trace ports
 - PECI interface 3.0
-- I2C voltage translator
-- 10 SMBUS headers
-- 4 SGPIO headers
+- 10 SMBUS instances
+- FAN, PMW and TACHO pins
 - VCI interface
-- 5 independent Hardware Driven PS/2 Ports
-- eSPI header
-- 3 Breathing/Blinking LEDs
-- 2 Sockets for SPI NOR chips
-- One reset and VCC_PWRDGD pushbuttons
-- One external PCA9555 I/O port with jumper selectable I2C address.
-- One external LTC2489 delta-sigma ADC with jumper selectable I2C address.
-- Board power jumper selectable from +5V 2.1mm/5.5mm barrel connector or USB Micro A connector.
+- Independent Hardware Driven PS/2 Ports
+
+At difference from MEC15xx evaluation board, modular MEC1501 exposes the pins
+in 2 different ways:
+
+1) Standalone mode via headers
+
+- GPIOs
```

Comment:
Almost... you forgot to indent these list items (3 spaces) so they show up under the numbered list item (see the recommended edit in the original comment)

Reply:
Done